lib/matplotlib/axis.py

```diff
@@ -2259,7 +2259,9 @@ def _init(self):
             transform=mtransforms.blended_transform_factory(
                 self.axes.transAxes, mtransforms.IdentityTransform()),
             fontsize=mpl.rcParams['xtick.labelsize'],
-            color=mpl.rcParams['xtick.color'],
+            color=mpl.rcParams['xtick.color'] if
```

Comment:
I find this is really hard to parse, and I'm surprised this passes out linter.  I would just make a `color` variable above this call to `set` and use it here.  Or barring that I'd indent the if/else somehow.  However, maybe this is standard?

Reply:
Thanks for the review, @jklymak 
I agree, this looks cluttered, I did not find clear PEP 8 instructions for ternary operations. I added the conditionals in the following commit before the `set` call as you suggested.